eng/config/globalconfigs/Common.globalconfig

```diff
@@ -4,7 +4,7 @@ dotnet_diagnostic.CA1067.severity = warning
 dotnet_diagnostic.CA1068.severity = warning
 dotnet_diagnostic.CA1200.severity = warning
 dotnet_diagnostic.CA1821.severity = warning
-dotnet_diagnostic.CA1865.severity = warning
+dotnet_diagnostic.CA1868.severity = warning
```

Comment:
In the future, we need to not check these in until the warning number is fixed. Another warning should not have come in and taken a number that had already effectively shipped to customers (us). I would prefer that we remove this entirely until the warning number is checked in.

Reply:
I agree, that was my fault, sorry about that. I should have waited until the PR was merged and **then** opened the PR to fix the warnings.
The analyzer has been merged, so the warning number is fixed.